<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Suzuki Lab.</title>
    <link rel="stylesheet" href="fontawesome/css/all.min.css"> <!-- https://fontawesome.com/ -->
    <link href="https://fonts.googleapis.com/css?family=Helvetica" rel="stylesheet" /> <!-- https://fonts.google.com/ -->
    <link rel="stylesheet" href="css/tooplate-wave-cafe.css">
<!--
Tooplate 2121 Wave Cafe
https://www.tooplate.com/view/2121-wave-cafe
-->
</head>
<body>
  <div class="tm-container">
    <div class="tm-row">
      <!-- Site Header -->
      <div class="tm-left">
        <div class="tm-left-inner">
          <div class="tm-site-header2">
            <div class="tm-site-header3">
            <i class="fas fa-dna fa-3x tm-site-logo"></i>
            <h1 class="tm-site-name">Suzuki Lab.</h1>
            </div>
            <div class="tm-site-header4">
            <p class="tm-site-name2">Division of Molecular Oncology<br>Nagoya University Graduate School of Medicine</p>
            </div>
          </div>
          <nav class="tm-site-nav">
            <ul class="tm-site-nav-ul">
              <li class="tm-page-nav-item">
                <a href="#about" class="tm-page-link active">
                  <i class="fas fa-brain tm-page-link-icon"></i>
                  <span>About us</span>
                </a>
              </li>
              <li class="tm-page-nav-item">
                <a href="#research" class="tm-page-link">
                  <i class="fas fa-microscope tm-page-link-icon"></i>
                  <span>Research lines</span>
                </a>
              </li>
              <li class="tm-page-nav-item">
                <a href="#news" class="tm-page-link">
                  <i class="fas fa-mug-hot tm-page-link-icon"></i>
                  <span>News</span>
                </a>
              </li>

              <li class="tm-page-nav-item">
                <a href="#member" class="tm-page-link">
                  <i class="fas fa-users tm-page-link-icon"></i>
                  <span>Members</span>
                </a>
              </li>
              <li class="tm-page-nav-item">
                <a href="#special" class="tm-page-link">
                  <i class="fas fa-book-reader tm-page-link-icon"></i>
                  <span>Publications</span>
                </a>
              </li>
              <li class="tm-page-nav-item">
                <a href="#recruit" class="tm-page-link">
                  <i class="fas fa-laugh tm-page-link-icon"></i>
                  <span>Join us!</span>
                </a>
              </li>

              <li class="tm-page-nav-item">
                <a href="#contact" class="tm-page-link">
                  <i class="fas fa-comments tm-page-link-icon"></i>
                  <span>Contact</span>
                </a>
              </li>
              <li class="tm-page-nav-item">
                <a href="#links" class="tm-page-link">
                  <i class="fas fa-comments tm-page-link-icon"></i>
                  <span>Link</span>
                </a>
              </li>
            </ul>
          </nav>
        </div>        
      </div>
      <div class="tm-right">
        <main class="tm-main">
          <div id="member" class="tm-page-content">
            <!-- About us -->
            <nav class="tm-black-bg tm-drinks-nav">
              <ul>
                <li>
                  <a href="#" class="tm-tab-link active" data-id="PI">PI</a>
                </li>
                <li>
                  <a href="#" class="tm-tab-link" data-id="hot">Researcher</a>
                </li>
                <li>
                  <a href="#" class="tm-tab-link" data-id="juice">Assistant</a>
                </li>
              </ul>
            </nav>

            <div id="PI" class="tm-tab-content">
              <div class="tm-list">
	              <div class="tm-black-bg tm-special-item">
	                <img src="img/hp_suzukihiroshi202005.jpeg" alt="Image" class="center">
	                <div class="tm-special-item-description">
	                  <h2 class="tm-text-primary tm-special-item-title">Hiroshi I. Suzuki, MD, PHD</h2>
	                  <p class="tm-special-item-text">2020<span>&#8211;</span> PresentNagoya University Professor<br> 
2019<span>&#8211;</span>2020 Massachusetts Institute of Technology Koch Institute for Integrative Cancer Research Research Scientist<br> 
2014<span>&#8211;</span>2019 Massachusetts Institute of Technology Koch Institute for Integrative Cancer Research Visiting Scientist<br> 
2010<span>&#8211;</span>2014 Graduate School of Medicine, The University of Tokyo Department of Molecular Pathology Project Assistant Professor<br> 
2008<span>&#8211;</span>2010 Japan Society for the Promotion of Science Research Fellowship for Young Scientists<br>
<a href="https://www.researchgate.net/profile/Hiroshi_Suzuki16">Researchgate</a><br>
<a href="https://researchmap.jp/hiroshisuzuki?lang=en">Researchmap</a>
<p>Email: hisuzuki@med.nagoya-u.ac.jp</p>  
	                </div>
	              </div>                                       
              </div>
            </div> 

            <div id="hot" class="tm-tab-content">
              <div class="tm-list">
              
                <div class="tm-list-item">          
                  <img src="img/hot-americano.png" alt="Image" class="tm-list-item-img">
                  <div class="tm-black-bg tm-list-item-text">
                    <h3 class="tm-list-item-name">Koichi Ogami, PHD<span class="tm-list-item-price">Assistant professor</span></h3>
                    <p class="tm-list-item-description"><a href="https://www.researchgate.net/profile/Koichi_Ogami">Researchgate</a><br>
<a href="https://researchmap.jp/koichi_ogami?lang=en">Researchmap</a><br>
<p>Email: koichi.ogami@med.nagoya-u.ac.jp</p>              
                  </div>
                </div>
                <div class="tm-list-item">          
                  <img src="img/hot-cappuccino.png" alt="Image" class="tm-list-item-img">
                  <div class="tm-black-bg tm-list-item-text">
                    <h3 class="tm-list-item-name">Koh Onimaru<span class="tm-list-item-price">Assistant professor</span></h3>
                    <p class="tm-list-item-description"><a href="https://www.researchgate.net/profile/Koh_Onimaru">Researchgate</a><br>
                    <a href="https://researchmap.jp/koonimaru/?lang=english">Researchmap</a></p><br>
                    <p><a href="https://sites.google.com/site/kohonimaru/home">Original page</a></p><br>
                    <p>Email: koh.onimaru@med.nagoya-u.ac.jp</p>                    
                  </div>
                </div>
              </div>
            </div>

            <div id="juice" class="tm-tab-content">
              <div class="tm-list">
                <div class="tm-list-item">          
                  <img src="img/smoothie-1.png" alt="Image" class="tm-list-item-img">
                  <div class="tm-black-bg tm-list-item-text">
                    <h3 class="tm-list-item-name">TBD</h3>
                    <p class="tm-list-item-description">TBD</p>              
                  </div>
                </div>              
              </div>
            </div>
            <!-- end Member Page -->
          </div>

          <!-- About Us Page -->
          <div id="about" class="tm-page-content">
            <div class="tm-black-bg tm-about">              
              <h2 class="tm-text-primary tm-about-header">About our lab</h2>
                <img src="img/about-1.png" alt="Image" class="center">
                <div class="tm-list-item-text">
                  <p>The mission of our laboratory is to elucidate the function and mechanism of genome and RNA and to apply them to better understand 
the molecular mechanisms of cancer and to develop the new approaches to cancer therapy. 
How do genes work? This fundamental question is crucial to understand the mechanisms of diverse diseases including cancer, 
which involve the interactions between genetic factors and environmental factors. 
Through an integrative omics analysis based on next-generation sequencing, bioinformatics, 
and genome/RNA engineering, we aim to get a deeper understanding of the relationships between genome, 
epigenome, transcription, and RNA to uncover the operating principles of gene regulation and disease mechanisms. 
We also aim to facilitate cancer precision medicine based on comprehensive omics analysis and develop new methodologies and technologies to advance this.</p>
              </div>
            </div>
          </div>
          <!-- end About Us Page -->
          <!-- research lines -->
          <div id="research" class="tm-page-content">

            <div class="tm-black-bg tm-about">              
                <div class="tm-list-item-text">
                  <h2 class="tm-text-primary">Research line 1: MicroRNA and Cancer</h2>
                    <img src="img/about-2.png" alt="Image" class="center">
                    <div class="tm-list-item-text">
                      <span class="span2" tabindex="0" >[Read]</span>
                      <span class="span3" tabindex="0">[Hide]</span>
                      <p class="hidden research">Function of many genes is governed by the flow of genetic information from DNA to RNA to protein, the central dogma of molecular biology. On the other hand, non-coding regions of the genome are pervasively transcribed and yield thousands of non-coding RNAs (ncRNAs). MicroRNAs (miRNAs) are abundant class of small ncRNAs with 21-25 nucleotides length. The first miRNA was discovered in 1993, and miRNA research was further accelerated by the Fire and Mello’s discovery of RNA interference (RNAi) in 1998.
In mammalian cells, the biogenesis of miRNAs is mediated by multiple biochemical steps including processing of miRNA precursors by two RNase III enzymes, Drosha and Dicer, and formation of RNA-induced silencing complex with Argonaute (Ago) proteins. miRNAs embedded in Ago proteins recognize their binding sequences predominantly located in 3’UTRs of target mRNAs through sequence complementarity between 5’ seed regions of miRNAs (nt 2-7) and their binding sites and suppress their expression. Reflecting the short sequence dependency, individual miRNAs regulate hundreds of target genes, thus forming complex gene regulatory networks. In addition to regulation of normal cells, miRNAs are known to modulate various pathological processes including cancer. We have investigated the relationships between cancer, miRNA biogenesis, and miRNA-mediated gene regulation and revealed the roles of tumor suppressor p53 in miRNA processing (Suzuki et al, Nature, 2009; Suzuki et al, Mol Cell, 2011).
In cancer biology, numerous studies have reported that miRNAs regulate many aspects of autonomous behavior of cancer cells, including sustained proliferation, resistance to cell death, and acquisition of metastasis phenotypes. We also previously reported that miRNAs regulate immune phenotypes of malignant lymphoma cells and modulate surrounding tumor microenvironments, illuminating miRNA-mediated non-cell-autonomous mechanism in cancer progression (Matsuyama et al, Blood, 2011; Suzuki et al, Oncogene, 2015).</p>                  
                    </div>                
                </div>
            </div>
            <div class="tm-black-bg tm-about">              
                <div class="tm-list-item-text">
                  <h2 class="tm-text-primary">Research line 2: Systems RNA Biology</h2>
                    <img src="img/about-3.png" alt="Image" class="center">
                    <div class="tm-list-item-text">
                      <span class="span2" tabindex="0" >[Read]</span>
                      <span class="span3" tabindex="0">[Hide]</span>

                      <p class="hidden research">Gene regulation by miRNAs is complicated with targeting of multiple mRNAs by 
individual miRNAs and their generally mild effects on target mRNAs. Understanding of such complex systems requires sophisticated approaches integrating multiple genome-wide datasets including transcriptomes. We have previously developed analytical approaches to infer miRNA activities from mRNA expression datasets and demonstrated that applying this approaches to analysis of paired miRNA-mRNA datasets 
improves identification of robust bio-markers in cancer (Suzuki et al, Nucleic Acid Res, 2013; Suzuki et al, Leukemia, 2013).
Of the two strands of miRNA duplexes, one miRNA strand is incorporated into an Ago protein, 
while the other strand is degraded. Asymmetric selection of one guide strand is crucial in miRNA-mediated gene regulation and specificity of RNAi. By combining molecular biology, biophysics, structural biology, and bioinformatics, we have presented a molecular and theoretical framework that describes the general features and underlying mechanisms of miRNA asymmetry and further presents modeling to estimate the asymmetry (Suzuki et al, Nat Struct Mol Biol, 2015; Matsuyama et al, IJMS, 2019). In consistent with the prediction by this model, cancer-associated miRNA variations were shown to modulate asymmetric selection. This model has contributed to development of synthetic RNA-Based immunomodulatory gene circuits for cancer immunotherapy (Nissim et al, Cell, 2017) and discovery of the first case of a pathogenic gain-of-function miRNA mutation in human diseases (Grigelioniene et al, Nature Medicine, 2019).</p>                  
                    </div>                
                </div>
            </div>
            <div class="tm-black-bg tm-about">              
                <div class="tm-list-item-text">
                  <h2 class="tm-text-primary">Research line 3: Biology of Transcription (1): Super-enhancer and Phase Separation</h2>
                    <img src="img/about-4.png" alt="Image" class="center">
                    <div class="tm-list-item-text">
                      <span class="span2" tabindex="0" >[Read]</span>
                      <span class="span3" tabindex="0">[Hide]</span>

                      <p class="hidden research">How do different cell types with the same genome establish cell-type-specific 
transcriptional programs? Cell-type-specific transcriptional programs are activated by the 
cell-type-specific enhancers bound by master transcription factors. Super-enhancers are 
emerging subclass of distal enhancers which regulate cell identity and disease processes. 
We have focused on the function of super-enhancers from the stand point of RNA. By integratively analyzing the relationships between super-enhancers and miRNAs, we have reported that super-enhancers have central roles in driving the function of cell-type-specific and evolutionally conserved miRNAs and maintaining their regulatory networks (Suzuki et al, Cell, 2017). In addition, we reported that super-enhancers facilitate processing of miRNA precursors, revealing unrecognized higher-order property of super-enhancers. Inspired from these findings, we currently instigate the roles of phase separation in gene regulation and enhancer control.</p>                  
                    </div>                
                </div>
            </div>
            <div class="tm-black-bg tm-about">              
                <div class="tm-list-item-text">
                  <h2 class="tm-text-primary">Research line 4: Biology of Transcription (2): Transcription Cycle</h2>
                    <img src="img/about-5.png" alt="Image" class="center">
                    <div class="tm-list-item-text">
                      <span class="span2" tabindex="0" >[Read]</span>
                      <span class="span3" tabindex="0">[Hide]</span>

                      <p class="hidden research">Toward better understanding of enhancer function, it is important to understand 
the dynamics of transcription across enhancers and the transcriptional impacts of enhancers on cognate promoters. 
Divergent transcription by RNA polymerase II occurs at many enhancers and promoters, yielding both coding mRNAs and 
diverse ncRNAs including eRNAs and uaRNAs. Furthermore, transcription by RNA polymerase II is regulated at multiple 
steps including initiation, pausing, pause release, elongation, and termination (“transcription cycle”). We have 
investigated the detailed mechanisms of transcription by integrating genome science and information science.<br>
Protein-coding promoters are characterized by elongation of RNA polymerase II in the sense direction and 
early termination in the antisense direction. Such transcriptional directionality is modulated by 
asymmetrical distribution of U1 splicing motif and poly-adenylation signal (PAS motif) (U1-PAS axis). 
We have previously reported that premature PAS termination also frequently occurs in the sense direction 
and coincides with RNA polymerase II pause sites associated with +1 stable nucleosomes 
(Chiu et al, Mol Cell, 2018). Thus, promoter proximal Pol II pausing consists of two processes: 
transcription start site (TSS)-proximal pausing and +1 stable nucleosome pausing.<br>
A recent advances in cancer genome studies have uncovered new class of cancer driver mutations affecting RNA processing and transcription. 
On the other hand, their functional significance has not been fully investigated in many cases. In myelodysplastic syndromes (MDS) 
and related neoplasms, the members of the cohesin complex including STAG2 are frequently mutated. In a recent collaboration study, 
we have importantly found that combined loss of STAG2 and master transcription factor RUNX1 synergistically attenuates enhancer–promoter 
loops, leading to development of MDS, and that cohesin deficiency preferentially downregulates genes with high basal transcriptional pausing, 
explaining the feature of selective gene dysregulation (Ochi et al, Cancer Discovery, 2020).</p>                  
                    </div>                
                </div>
            </div>
            <div class="tm-black-bg tm-about">              
                <div class="tm-list-item-text">
                  <h2 class="tm-text-primary">Research line 5: Multi-Omics Analysis in Cancer</h2>
                    <img src="img/about-6.png" alt="Image" class="center">
                    <div class="tm-list-item-text">
                      <span class="span2" tabindex="0" >[Read]</span>
                      <span class="span3" tabindex="0">[Hide]</span>

                      <p class="hidden research">We also aim to identify new molecular targets in cancer therapies through integrative omics analysis. In a previous study, by comparing super-enhancers in normal cells/tissues and cancer cells, we have reported that gain and loss of super-enhancers are frequently found in the neighborhoods of oncogenic and tumor-suppressive miRNAs associated with various cancer hallmark traits (Suzuki et al, Cell, 2017). Furthermore, we have recently reported the first case of miRNA gain-of-function mutation in human disease (Grigelioniene et al, Nature Medicine, 2019), and in a companion study, we performed systematic analysis of siRNA off-target data and genome-wide RNAi cancer lethality screens using 501 human cancer cell lines, a cancer dependency map, and offered a basis for improvement of RNAi-based functional genomics (Suzuki et al, Nature Genetics, 2018).</p>                  
                    </div>                
                </div>
            </div>
            <div class="tm-black-bg tm-about">              
                <div class="tm-list-item-text">
                  <h2 class="tm-text-primary">Beyond</h2>
                    <img src="img/about-7.png" alt="Image" class="center">
                    <div class="tm-list-item-text">
                      <p></p>                  
                    </div>                
                </div>
            </div>
          </div>

          <!--end research lines -->
          <!--News page-->
          <div id="news" class="tm-page-content">

            <div class="tm-black-bg tm-about">              
                <div class="tm-list-item-text">
                  <h2 class="tm-text-primary"><dl><dt>2020/12/01</dt><dd>Koh and Koichi joined the lab!</dd></dl></h2>
                </div>
            </div>
          </div>
          <!--end News-->


          <!-- Publication list -->
          <div id="special" class="tm-page-content">
            <div class="tm-special-items">
              <div class="tm-black-bg tm-about">
                <!-- <img src="img/special-04.jpg" alt="Image" class="center"> -->
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">Selected publications</h2>
                  <p class="tm-special-item-text">
                    <ol>
                      <li>
Ochi Y, Kon A, ..(31 authors).. Suzuki HI, Ogawa S. Combined Cohesin-RUNX1 Deficiency Synergistically Perturbs Chromatin Looping and Causes Myelodysplastic Syndromes. <a href=https://doi.org/10.1158/2159-8290.CD-19-0982>Cancer Discov. 2020 Jun;10(6):836-853.</a>
                      </li>
                      <li>
Grigelioniene G, Suzuki HI, Taylan F, Mirzamohammadi F, Borochowitz ZU, Ayturk UM, Tzur S, Horemuzova E, Lindstrand A, Weis MA, Grigelionis G, Hammarsjö A, Marsk E, Nordgren A, Nordenskjöld M, Eyre DR, Warman ML, Nishimura G, Sharp PA, Kobayashi T. Gain-of-function mutation of microRNA-140 in human skeletal dysplasia. <a href=https://doi.org/10.1038/s41591-019-0353-2>Nat Med. 2019 Apr;25(4):583-590. Epub 2019 Feb 25.</a>
                      </li>
                      <li>
Suzuki HI, Spengler RM, Grigelioniene G, Kobayashi T, Sharp PA. Deconvolution of seed and RNA-binding protein crosstalk in RNAi-based functional genomics. <a href=https://doi.org/10.1038/s41588-018-0104-1>Nat Genet. 2018; 50(5): 657-661.</a>
                      </li>
                      <li>
Chiu AC, Suzuki HI, Wu X, Mahat DB, Kriz AJ, Sharp PA. Transcriptional Pause Sites Delineate Stable Nucleosome-Associated Premature Polyadenylation Suppressed by U1 snRNP. <a href=https://doi.org/10.1016/j.molcel.2018.01.006>Mol Cell. 2018; 15; 69(4): 648-663.e7.</a>
                      </li> 
                      <li>
Suzuki HI, Young RA, Sharp PA. Super-Enhancer-Mediated RNA Processing Revealed by Integrative MicroRNA Network Analysis. <a href=https://doi.org/10.1016/j.cell.2017.02.015>Cell, 2017; 168(6): 1000-1014.</a>
                      </li>
                      <li>
Suzuki HI, Katsura A, Yasuda T, Ueno T, Mano H, Sugimoto K, Miyazono K. Small-RNA asymmetry is directly driven by mammalian Argonautes. <a href=https://www.nature.com/articles/nsmb.3050?proof=t>Nat. Struct. Mol. Biol. 2015; 22(7): 512-521.</a>
                      </li>
                      <li>
Suzuki HI, Matsuyama H, Noguchi M, Yao T, Komatsu N, Mano H, Sugimoto K, Miyazono K. Computational dissection of distinct microRNA activity signatures associated with peripheral T cell lymphoma subtypes. <a href=https://www.nature.com/articles/leu2013121>Leukemia, 2013; 27(10): 2107-2111.</a>
                      </li>
                      <li>
Suzuki HI, Mihira H, Watabe T, Sugimoto K, Miyazono K. Widespread inference of weighted microRNA-mediated gene regulation in cancer transcriptome analysis. <a href=https://doi.org/10.1093/nar/gks1439>Nucleic Acids Res. 2013; 1; 41(5): e62.</a>
                      </li>
                     <li>
Suzuki HI, Arase M, Matsuyama H, Choi YL, Ueno T, Mano H, Sugimoto K, Miyazono K. MCPIP1 ribonuclease antagonizes dicer and terminates microRNA biogenesis through precursor microRNA degradation. <a href=https://doi.org/10.1016/j.molcel.2011.09.012>Mol Cell, 2011; 44(3): 424-436.</a>
                      </li>
                     <li>
Matsuyama H, Suzuki HI, Nishimori H, Noguchi M, Yao T, Komatsu N, Mano H, Sugimoto K, Miyazono K. miR-135b mediates NPM-ALK-driven oncogenicity and renders IL-17-producing immunophenotype to anaplastic large cell lymphoma. <a href=https://doi.org/10.1182/blood-2011-05-3546542>Blood. 2011; 22; 118(26): 6881-92.</a>
                      </li>
                      <li>
Suzuki HI, Yamagata K, Sugimoto K, Iwamoto T, Kato S, Miyazono K. Modulation of microRNA processing by p53. <a href=https://www.nature.com/articles/nature08199>Nature, 2009; 460(7254): 529-533.</a>
                      </li>
                    </ol>
                  </p>
                </div>
              </div>

              <div class="tm-black-bg tm-about">
                <img src="img/special-01.png" alt="Image" class="center">
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">2020</h2>
                  <p class="tm-special-item-text">
                    <ol>
                      <li>
Ochi Y, Kon A, Sakata T, Nakagawa MM, Nakazawa N, Kakuta M, Kataoka K, Koseki H, Nakayama M, Morishita D, Tsuruyama T, Saiki R, Yoda A, Okuda R, Yoshizato T, Yoshida K, Shiozawa Y, Nannya Y, Kotani S, Kogure Y, Kakiuchi N, Nishimura T, Makishima H, Malcovati L, Yokoyama A, Takeuchi K, Sugihara E, Sato TA, Sanada M, Takaori-Kondo A, Cazzola M, Kengaku M, Miyano S, Shirahige K, Suzuki HI, Ogawa S. Combined Cohesin-RUNX1 Deficiency Synergistically Perturbs Chromatin Looping and Causes Myelodysplastic Syndromes. <a href=https://doi.org/10.1158/2159-8290.CD-19-0982>Cancer Discov. 2020 Jun;10(6):836-853.</a>
                      </li>
                      <li>
Tomofuji Y, Takaba H, Suzuki HI, Benlaribi R, Martinez CDP, Abe Y, Morishita Y, Okamura T, Taguchi A, Kodama T, Takayanagi H. Chd4 choreographs self-antigen expression for central immune tolerance. <a href=https://www.nature.com/articles/s41590-020-0717-2>Nat Immunol. 2020 Jun 29. Online ahead of print.</a>
                      </li>
                      <li>
Ochi Y, Kon A, Sakata T, Nakagawa MM, Nakazawa N, Kakuta M, Kataoka K, Koseki H, Nakayama M, Morishita D, Tsuruyama T, Saiki R, Yoda A, Okuda R, Yoshizato T, Yoshida K, Shiozawa Y, Nannya Y, Kotani S, Kogure Y, Kakiuchi N, Nishimura T, Makishima H, Malcovati L, Yokoyama A, Takeuchi K, Sugihara E, Sato TA, Sanada M, Takaori-Kondo A, Cazzola M, Kengaku M, Miyano S, Shirahige K, Suzuki HI, Ogawa S. Combined Cohesin-RUNX1 Deficiency Synergistically Perturbs Chromatin Looping and Causes Myelodysplastic Syndromes. <a href=https://cancerdiscovery.aacrjournals.org/content/10/6/836>Cancer Discov, 2020; 10(6): 836-853.</a>
                      </li>
                    </ol>
                  </p>
                </div>
              </div>
              <div class="tm-black-bg tm-about">
                <img src="img/special-02.png" alt="Image" class="center">
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">2019</h2>
                    <ol>
                      <li>
Matsuyama H, Suzuki HI. Systems and Synthetic microRNA Biology: From Biogenesis to Disease Pathogenesis. <a href=https://www.mdpi.com/1422-0067/21/1/132>Int J Mol Sci, 2019; 21(1): 132.</a>
                      </li>
                      <li>
Tominaga K, Suzuki HI. TGF-β Signaling in Cellular Senescence and Aging-Related Pathology. <a href=https://www.mdpi.com/1422-0067/20/20/5002>Int J Mol Sci, 2019; 20(20): 5002.</a>
                      </li>
                      <li>
Kelly TJ, Suzuki HI, Zamudio JR, Suzuki M, Sharp PA. Sequestration of microRNA-mediated target repression by the Ago2-associated RNA-binding protein FAM120A. <a href=https://rnajournal.cshlp.org/content/early/2019/07/09/rna.071621.119?top=1>RNA, 2019; 25(10): 1291-1297.</a>
                      </li>
                      <li>
Akatsu Y, Takahashi N, Yoshimatsu Y, Kimuro S, Muramatsu T, Katsura A, Maishi N, Suzuki HI, Inazawa J, Hida K, Miyazono K, Watabe T. Fibroblast growth factor signals regulate transforming growth factor-β-induced endothelial-to-myofibroblast transition of tumor endothelial cells via Elk1. <a href=https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1878-0261.12504>Mol Oncol, 2019; 13(8): 1706-1724.</a>
                      </li>
                      <li>
Grigelioniene G, Suzuki HI, Taylan F, Mirzamohammadi F, Borochowitz ZU, Ayturk UM, Tzur S, Horemuzova E, Lindstrand A, Weis MA, Grigelionis G, Hammarsjö A, Marsk E, Nordgren A, Nordenskjöld M, Eyre DR, Warman ML, Nishimura G, Sharp PA, Kobayashi T. Gain-of-function mutation of microRNA-140 in human skeletal dysplasia. <a href=https://www.nature.com/articles/s41591-019-0353-2>Nat Med, 2019; 25(4): 583-590.</a>
                      </li>

                    </ol>
                </div>
              </div>
              <div class="tm-black-bg tm-about">
                <img src="img/special-03.png" alt="Image" class="center">
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">2018</h2>
                    <ol>
                      <li>
Souma K, Shichino S, Hashimoto S, Ueha S, Tsukui T, Nakajima T, Suzuki HI, Shand FHW, Inagaki Y, Nagase T, Matsushima K. Lung fibroblasts express a miR-19a-19b-20a sub-cluster to suppress TGF-β-associated fibroblast activation in murine pulmonary fibrosis. <a href=https://www.nature.com/articles/s41598-018-34839-0>Sci Rep, 2018; 8(1): 16642.</a>
                      </li>
                      <li>
Gao A, Shrinivas K, Lepeudry P, Suzuki HI, Sharp PA, Chakraborty AK. Evolution of weak cooperative interactions for biological specificity. <a href=https://doi.org/10.1073/pnas.1815912115>Proc Natl Acad Sci U S A, 2018; 115(47): E11053-E11060.</a>
                      </li>
                      <li>
Suzuki HI, Horie M, Mihira H, Saito A. Molecular Analysis of Endothelial-mesenchymal Transition Induced by Transforming Growth Factor-β Signaling. <a href=https://www.jove.com/v/57577/molecular-analysis-endothelial-mesenchymal-transition-induced>J Vis Exp, 2018; (138): 57577.</a>
                      </li>
                      <li>
Miyashita N, Horie M, Suzuki HI, Yoshihara M, Djureinovic D, Persson J, Brunnström H, Lindskog C, Elfving H, Micke P, Saito A, Nagase T. An Integrative Analysis of Transcriptome and Epigenome Features of ASCL1-Positive Lung Adenocarcinomas. <a href=https://doi.org/10.1016/j.jtho.2018.07.096>J Thorac Oncol, 2018; 13(11): 1676-1691.</a>
                      </li>
                      <li>
Suzuki HI. MicroRNA Control of TGF-β Signaling. <a href=https://doi.org/10.3390/ijms19071901>Int J Mol Sci, 2019; 19(7): 1901.</a>
                      </li>
                      <li>
Suzuki HI, Spengler RM, Grigelioniene G, Kobayashi T, Sharp PA. Deconvolution of seed and RNA-binding protein crosstalk in RNAi-based functional genomics. <a href=https://www.nature.com/articles/s41588-018-0104-1>Nat Genet, 2018; 50(5): 657-661.</a>
                      </li>

                    </ol>

                </div>
              </div>
              <div class="tm-black-bg tm-about">
                <img src="img/special-04.png" alt="Image" class="center">
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">2017</h2>
                    <ol>
                      <li>
Nissim L, Wu MR, Pery E, Binder-Nissim A, Suzuki HI, Stupp D, Wehrspaun C, Tabach Y, Sharp PA, Lu TK. Synthetic RNA-Based Immunomodulatory Gene Circuits for Cancer Immunotherapy. <a href=https://doi.org/10.1016/j.cell.2017.09.049>Cell, 2017; 171(5): 1138-1150.</a>
                      </li>
                      <li>
Suzuki HI, Katsura A, Mihira H, Horie M, Saito A, Miyazono K. Regulation of TGF-β-mediated endothelial-mesenchymal transition by microRNA-27. <a href=https://doi.org/10.1093/jb/mvx017>J Biochem, 2017; 161(5): 417-420.</a>
                      </li>
                      <li>
Suzuki HI, Young RA, Sharp PA. Super-Enhancer-Mediated RNA Processing Revealed by Integrative MicroRNA Network Analysis. <a href=https://doi.org/10.1016/j.cell.2017.02.015>Cell, 2017; 168(6): 1000-1014.</a>
                      </li>
                    </ol>

                </div>
              </div>
              <div class="tm-black-bg tm-about">
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">2016</h2>
                    <ol>
                      <li>
Horie M, Saito A, Ohshima M, Suzuki HI, Nagase T. YAP and TAZ modulate cell phenotype in a subset of small cell lung cancer. <a href=https://doi.org/10.1111/cas.13078>Cancer Sci, 2016; 107(12): 1755-1766.</a>
                      </li>
                      <li>
Katsura A, Suzuki HI, Ueno T, Mihira H, Yamazaki T, Yasuda T, Watabe T, Mano H, Yamada Y, Miyazono K. MicroRNA-31 is a positive modulator of endothelial-mesenchymal transition and associated secretory phenotype induced by TGF-β. <a href=https://doi.org/10.1111/gtc.12323>Genes Cells, 2016; 21(1): 99-116.</a>
                      </li>
                    </ol>

                </div>
              </div>
              <div class="tm-black-bg tm-about">
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">2015</h2>
                    <ol>
                      <li>
Shichino S, Abe J, Ueha S, Otsuji M, Tsukui T, Kosugi-Kanaya M, Shand FH, Hashimoto S, Suzuki HI, Morikawa T, Inagaki Y, Matsushima K. Reduced supply of monocyte-derived macrophages leads to a transition from nodular to diffuse lesions and tissue cell activation in silica-induced pulmonary fibrosis in mice. <a href=https://doi.org/10.1016/j.ajpath.2015.07.013>Am J Pathol, 2015; 185(11): 2923-2938.</a>
                      </li>
                      <li>
Sugimoto K, Suzuki HI, Fujimura T, Ono A, Kaga N, Isobe Y, Sasaki M, Taka H, Miyazono K, Komatsu N. A clinically attainable dose of L-asparaginase targets glutamine addiction in lymphoid cell lines. <a href=https://doi.org/10.1111/cas.12807>Cancer Sci, 2015; 106(11): 1534-1543.</a>
                      </li>
                      <li>
Suzuki HI, Katsura A, Yasuda T, Ueno T, Mano H, Sugimoto K, Miyazono K. Small-RNA asymmetry is directly driven by mammalian Argonautes. <a href=https://www.nature.com/articles/nsmb.3050?proof=t>Nat Struct Mol Biol, 2015; 22(7): 512-521.</a>
                      </li>
                      <li>
Suzuki HI, Katsura A, Miyazono K. A role of uridylation pathway for blockade of let-7 microRNA biogenesis by Lin28B. <a href=https://dx.doi.org/10.1111%2Fcas.12721>Cancer Sci, 2015; 106(9): 1174-1181.</a>
                      </li>
                      <li>
Itami S, Eguchi Y, Mizutani T, Aoki E, Ohgi T, Kuroda M, Ochiya T, Kato N, Suzuki HI, Kawada N, Murakami Y. Control of HCV Replication With iMIRs, a Novel Anti-RNAi Agent. <a href=https://doi.org/10.1038/mtna.2014.71>Mol Ther Nucleic Acids, 2015; 4(1): e219.</a>
                      </li>
                      <li>
Suzuki HI, Katsura A, Matsuyama H, Miyazono K. MicroRNA regulons in tumor microenvironment. <a href=https://doi.org/10.1038/onc.2014.254>Oncogene, 2015; 34(24): 3085-3094.</a>
                      </li>

                    </ol>

                </div>
              </div>
              <div class="tm-black-bg tm-about">
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">2014</h2>
                    <ol>
                      <li>
Horie M, Saito A, Noguchi S, Yamaguchi Y, Ohshima M, Morishita Y, Suzuki HI, Kohyama T, Nagase T. Differential knockdown of TGF-β ligands in a three-dimensional co-culture tumor- stromal interaction model of lung cancer. <a href=https://bmccancer.biomedcentral.com/articles/10.1186/1471-2407-14-580>BMC Cancer, 2014; 14: 580.</a>
                      </li>
                      <li>
Noguchi S, Saito A, Horie M, Mikami Y, Suzuki HI, Morishita Y, Ohshima M, Abiko Y, Mattsson JS, König H, Lohr M, Edlund K, Botling J, Micke P, Nagase T. An integrative analysis of the tumorigenic role of TAZ in human non-small cell lung cancer. <a href=https://clincancerres.aacrjournals.org/content/20/17/4660>Clin Cancer Res, 2014; 20(17): 4660-4672.</a>
                      </li>
                      <li>
Tanaka M, Suzuki HI, Shibahara J, Kunita A, Isagawa T, Yoshimi A, Kurokawa M, Miyazono K, Aburatani H, Ishikawa S, Fukayama M. EVI1 oncogene promotes KRAS pathway through suppression of microRNA-96 in pancreatic carcinogenesis. <a href=https://www.nature.com/articles/onc2013204>Oncogene, 2014; 33(19): 2454-2463.</a>
                      </li>
                    </ol>
                </div>
              </div>
              <div class="tm-black-bg tm-about">
                <div class="tm-special-item-description">
                  <h2 class="tm-text-primary tm-special-item-title">2013</h2>
                    <ol>
                      <li>
Yoshimatsu Y, Lee YG, Akatsu Y, Taguchi L, Suzuki HI, Cunha SI, Maruyama K, Suzuki Y, Yamazaki T, Katsura A, Oh SP, Zimmers TA, Lee SJ, Pietras K, Koh GY, Miyazono K, Watabe T. Bone morphogenetic protein-9 inhibits lymphatic vessel formation via activin receptor-like kinase 1 during development and cancer progression. <a href=https://doi.org/10.1073/pnas.1310479110>Proc Natl Acad Sci U S A, 2013; 110(47): 18940-18945.</a>
                      </li>
                      <li>
Suzuki HI, Matsuyama H, Noguchi M, Yao T, Komatsu N, Mano H, Sugimoto K, Miyazono K. Computational dissection of distinct microRNA activity signatures associated with peripheral T cell lymphoma subtypes. <a href=https://www.nature.com/articles/leu2013121>Leukemia, 2013; 27(10): 2107-2111.</a>
                      </li>
                      <li>
Saito A, Suzuki HI, Horie M, Ohshima M, Morishita Y, Abiko Y, Nagase T. An integrated expression profiling reveals target genes of TGF-β and TNF-α possibly mediated by microRNAs in lung cancer cells. <a href=https://doi.org/10.1371/journal.pone.0056587>PLoS One, 2013; 8(2): e56587.</a>
                      </li>
                      <li>
Suzuki HI, Mihira H, Watabe T, Sugimoto K, Miyazono K. Widespread inference of weighted microRNA-mediated gene regulation in cancer transcriptome analysis. <a href=https://doi.org/10.1093/nar/gks1439>Nucleic Acids Res, 2013; 41(5): e62.</a>
                      </li>
                      <li>
Suzuki HI, Miyazono K. p53 actions on microRNA expression and maturation pathway. <a href=https://doi.org/10.1007/978-1-62703-236-0_14>Methods Mol Biol, 2013; 962: 165-181.</a>
                      </li>
                      <li>
Liang C, Xiong K, Szulwach KE, Zhang Y, Wang Z, Peng J, Fu M, Jin P, Suzuki HI, Liu Q.. Sjogren syndrome antigen B (SSB)/La promotes global microRNA expression by binding microRNA precursors through stem-loop recognition. <a href=https://doi.org/10.1074/jbc.m112.401323>J Biol Chem, 2013; 288(1): 723-736.</a>
                      </li>
                    </ol>
                </div>
              </div>
            </div>            
          </div>
          <!-- end Publication list -->
          <!-- Recruit -->
          <div id="recruit" class="tm-page-content">
            <div class="tm-black-bg tm-about">
              <div class="tm-list-item-text">
              <h2 class="tm-text-primary">Join us!</h2>
              <div class="tm-list-item-text">
              <p>We are recruiting staff and graduate students from all over the world. Please send your curriculum vitae, bibliography, and a letter describing your research or clinical experience by e-mail to us.
              </div>
              </div>
            </div>
          </div>
          <!-- end Recruit Page -->

          <!-- Contact Page -->
          <div id="contact" class="tm-page-content">
            <div class="tm-black-bg tm-about">
              <div class="tm-list-item-text">
              <h2 class="tm-text-primary">Contact</h2>
              <div class="tm-list-item-text">
              <p>Lab address:
Nagoya University Medical Science Research Building 3, 505-1
65 Tsurumai-cho, Showa-ku
Nagoya city
Japan
Postal code: 466-8550</p>
			<p>Tel: +81-52-744-2460</p>
			<p>Email: hisuzuki@med.nagoya-u.ac.jp</p>
			</div>
            </div>
            </div>
          </div>
          <!-- end Contact Page -->
          <!-- Contact Page -->
          <div id="links" class="tm-page-content">
            <div class="tm-black-bg tm-about">
              <div class="tm-list-item-text">
              <h2 class="tm-text-primary">Links</h2>
              <div class="tm-list-item-text">
              <p><a href=https://www.med.nagoya-u.ac.jp/medical_E/laboratory/basic-med/oncology/mol-oncology target="_blank" rel="noopener noreferrer">Lab web page on Nagoya Univercity Graduate School of Medicine</a></p>
              <p><a href=https://www.med.nagoya-u.ac.jp/medical_E target="_blank" rel="noopener noreferrer">Nagoya Univercity Graduate School of Medicine</a></p>
              <p><a href=http://en.nagoya-u.ac.jp target="_blank" rel="noopener noreferrer">Nagoya Univercity</a></p>
			</div>
            </div>
            </div>
          </div>
          <!-- end Contact Page -->
        </main>
      </div>    
    </div>
  </div>
    
  <!-- Background video -->
  <div class="tm-video-wrapper">
      <i id="tm-video-control-button" class="fas fa-pause"></i>
      <video autoplay muted loop id="tm-video">
          <source src="video/water.m4v" type="video/mp4">
      </video>
  </div>

  <script src="js/jquery-3.4.1.min.js"></script>    
  <script>

    function setVideoSize() {
      const vidWidth = 1920;
      const vidHeight = 1080;
      const windowWidth = window.innerWidth;
      const windowHeight = window.innerHeight;
      const tempVidWidth = windowHeight * vidWidth / vidHeight;
      const tempVidHeight = windowWidth * vidHeight / vidWidth;
      const newVidWidth = tempVidWidth > windowWidth ? tempVidWidth : windowWidth;
      const newVidHeight = tempVidHeight > windowHeight ? tempVidHeight : windowHeight;
      const tmVideo = $('#tm-video');

      tmVideo.css('width', newVidWidth);
      tmVideo.css('height', newVidHeight);
    }

    function openTab(evt, id) {
      $('.tm-tab-content').hide();
      $('#' + id).show();
      $('.tm-tab-link').removeClass('active');
      $(evt.currentTarget).addClass('active');
    }    

    function initPage() {
      let pageId = location.hash;

      if(pageId) {
        highlightMenu($(`.tm-page-link[href^="${pageId}"]`)); 
        showPage($(pageId));
      }
      else {
        pageId = $('.tm-page-link.active').attr('href');
        showPage($(pageId));
      }
    }

    function highlightMenu(menuItem) {
      $('.tm-page-link').removeClass('active');
      menuItem.addClass('active');
    }

    function showPage(page) {
      $('.tm-page-content').hide();
      page.show();
    }

    $(document).ready(function(){

      /***************** Pages *****************/

      initPage();

      $('.tm-page-link').click(function(event) {
        
        if(window.innerWidth > 991) {
          event.preventDefault();
        }

        highlightMenu($(event.currentTarget));
        showPage($(event.currentTarget.hash));
      });

      
      /***************** Tabs *******************/

      $('.tm-tab-link').on('click', e => {
        e.preventDefault(); 
        openTab(e, $(e.target).data('id'));
      });

      $('.tm-tab-link.active').click(); // Open default tab


      /************** Video background *********/

      setVideoSize();

      // Set video background size based on window size
      let timeout;
      window.onresize = function(){
        clearTimeout(timeout);
        timeout = setTimeout(setVideoSize, 100);
      };

      // Play/Pause button for video background      
      const btn = $("#tm-video-control-button");

      btn.on("click", function(e) {
        const video = document.getElementById("tm-video");
        $(this).removeClass();

        if (video.paused) {
          video.play();
          $(this).addClass("fas fa-pause");
        } else {
          video.pause();
          $(this).addClass("fas fa-play");
        }
      });
    });
      
  </script>
</body>
</html>